charts/flyte-core/values-onprem.yaml

```diff
@@ -0,0 +1,841 @@
+    webpackHMR: false
+    # --- separateGrpcIngress puts GRPC routes into a separate ingress if true. Required for certain ingress controllers like nginx.
+    separateGrpcIngress: true
+    # --- Extra Ingress annotations applied only to the GRPC ingress. Only makes sense if `separateGrpcIngress` is enabled.
+    separateGrpcIngressAnnotations:
+      nginx.ingress.kubernetes.io/backend-protocol: "GRPC"
+    # --- Ingress annotations applied to both HTTP and GRPC ingresses.
+    annotations:
+      nginx.ingress.kubernetes.io/app-root: /console
+    # --- albSSLRedirect adds a special route for ssl redirect. Only useful in combination with the AWS LoadBalancer Controller.
+    albSSLRedirect: false
+    # --- Ingress hostname
+    host: #put ingress host name here
+    tls:
+      enabled: true
+      # --- Override default secret containing the tls certificate
+      secretName: #put ingress tls here
+  flyteNamespaceTemplate:
+    # --- Enable or disable creating Flyte namespace in template. Enable when using helm as template-engine only. Disable when using `helm install ...`.
+    enabled: false
+
+# -----------------------------------------------------
+# Core dependencies that should be configured for Flyte to work on any platform
+# Specifically 2 - Storage (s3, gcs etc), Production RDBMS - Aurora, CloudSQL etc
+# ------------------------------------------------------
+#
+# STORAGE SETTINGS
+#
+
+storage:
+  # -- Sets the storage type. Supported values are sandbox, s3, gcs and custom.
+  type: s3
+  # -- bucketName defines the storage bucket flyte will use. Required for all types except for sandbox.
+  bucketName: flytedata
+  # -- settings for storage type s3
+  s3:
+    region: us-east-1
+    endpoint: ""
+    # -- type of authentication to use for S3 buckets, can either be iam or accesskey
+    authType: accesskey
+    # -- AWS IAM user access key ID to use for S3 bucket auth, only used if authType is set to accesskey
+    accessKey: ""
+    # -- AWS IAM user secret access key to use for S3 bucket auth, only used if authType is set to accesskey
+    secretKey: ""
+  # -- settings for storage type gcs
+  gcs:
+  # -- GCP project ID. Required for storage type gcs.
+  # projectId:
+  # -- Settings for storage type custom. See https://github:com/graymeta/stow for supported storage providers/settings.
+  custom: {}
+  # -- toggles multi-container storage config
+  enableMultiContainer: false
+  # -- default limits being applied to storage config
+  limits:
+    maxDownloadMBs: 10
+
+# Database configuration(These are the values for a pgdb instance with hostname of postgres-flyte and postgres/password creds)
+db:
+  datacatalog:
+    database:
+      port: 5432
+      username: postgres
+      password: password
+      host: postgres-flyte
+      dbname: "datacatalog"
+  admin:
+    database:
+      port: 5432
+      username: postgres
+      password: password
+      host: postgresflyte
```

Comment:
Are you using different host for admin and datacatalog (L399)

Reply:
No its the same. Let me reconfirm